bpf/process/bpf_generic_kprobe.c

```diff
@@ -302,3 +302,17 @@ generic_kprobe_override(void *ctx)
 	map_delete_elem(&override_tasks, &id);
 	return 0;
 }
+
+__attribute__((section("fmod_ret/security_task_prctl"), used)) long
```

Comment:
I guess we do not always attach this to security_task_prctl, correct? Can we remove the section, or is it needed?

Reply:
it's needed for the `contrib/verify/verify.sh` script to pass, so it needs to stay, but I forgot to add some comments about that, will add